DOCSUP-5874: Edit and translate to Russian (Add Recipes Dataset) (#21702)
ClickHouse/ClickHouse: docs/en/getting-started/example-datasets/recipes.md

-   File is using only double quotes to enclose CSV strings; some strings are not enclosed in double quotes, and single quote must not be parsed as the string enclosing - that's why we also add the `--format_csv_allow_single_quote 0` parameter;
-   Some strings from CSV cannot parse, because they contain `\M/` sequence at the beginning of the value; the only value starting with backslash in CSV can be `\N` that is parsed as SQL NULL. We add `--input_format_allow_errors_num 10` parameter and up to ten malformed records can be skipped;
-   There are arrays for ingredients, directions and NER fields; these arrays are represented in unusual form: they are serialized into string as JSON and then placed in CSV - we parse them as String and then use [JSONExtract](../../sql-reference/functions/json-functions/) function to transform it to Array.

## Validate the Inserted Data

By checking the row count:

Query:

``` sq;
SELECT count() FROM recipes;
```

Result:

``` text
┌─count()─┐
│ 2231141 │
└─────────┘
```

## Example Queries

### Top Components by the Number of Recipes:

> In this example we learn how to use [arrayJoin](../../sql-reference/functions/array-join/) function to multiply data by array elements.

Reviewer: ```suggestion
In this example we learn how to use [arrayJoin](../../sql-reference/functions/array-join/) function to expand an array into a set of rows.
```
